cvat/apps/engine/errors.py

```diff
@@ -0,0 +1,15 @@
+from django.conf import settings
```

Comment:
Need to add the copyright header

Reply:
Can we merge iam/exceptions.py and engine/errors.py? I would remove the file.

Reply:
Done